frontend/views/containers/navigation/Profile.vue

```diff
@@ -38,9 +38,10 @@ export default ({
   },
   computed: {
     ...mapGetters([
-      'ourUsername'
+      'ourUsername',
+      'ourIdentityContractId'
     ]),
-    userDisplayName () {
+    userDisplayNameFromID () {
```

Comment:
Why was this renamed to `userDisplayNameFromID`? This is not a function that takes an ID... the previous name seems fine